docs/service-normalizationservice.html

```
@@ -185,10 +494,124 @@
+    Corrects resource IDs that are improperly formatted, such as IDs exceeding 64 characters.
+    
+    **Use Case**: Complying with FHIR standards and maintaining reference consistency.
+    
+    **Details**:
+    Replaces the invalid ID with a **hash** of the original ID.
+    Updates any **references** to the corrected ID to ensure integrity.
+  </Accordion>
+  <Accordion title="Copy Elements">
+    Copies values between properties in a FHIR resource using FHIRPath expressions.
+<p><strong>Use Case</strong>: Populating properties required for measure evaluation or reporting.</p>
+<p><strong>Details</strong>:
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 571.

(tag-pair)